style.css

```diff
@@ -113,3 +113,146 @@ header {
   list-style-type: none;
   padding-bottom: 5rem;
 }
+
+.projects {
+  margin: 20% 6% 10% 6%;
+}
+
+.card {
+  background-color: #fff;
+  border-radius: 16px;
+  padding: 16px;
+}
+
+.card-title {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  margin-bottom: 7px;
+}
+
+.project-experience {
+  display: flex;
+  font-size: 13px;
+  align-content: start;
+  gap: 30px;
+  margin-bottom: 20px;
+}
+
+.first-list {
+  list-style: none;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.other-list {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.project-summary {
+  line-height: 24px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  color: #344563;
+  margin-bottom: 10px;
+}
+
+.projects .card {
+  margin-bottom: 35px;
+}
+
+.tech-stack {
+  display: flex;
+  gap: 10px;
+  margin-bottom: 10px;
+}
+
+.tech-stack li {
+  list-style: none;
+  padding: 4px 12px;
+  background: #ebebff;
+  color: #6070ff;
+  letter-spacing: 0.03em;
+  border-radius: 8px;
+}
+
+.btn-primary-outline {
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you add the different button interactions including hover, active (when it's pressed), disabled, and enable (default state) as shown in the Figma template below:


![Screenshot (248)](https://user-images.githubusercontent.com/78104544/235187165-4f3535fb-2393-4aee-9e4a-1184ba22bde2.png)

the button on the pressed state has the background:  #2230D2; and on hover has the background: #6070FF;

**Example:**

**For pressed state**
```
button:active {
   background-color: #2230D2;
}
```

**For hover state**

```
button:hover {
    background-color: #6070FF;
}
```

Reply:
modified the button states